experiment_impact_tracker/utils.py

```diff
@@ -140,7 +148,11 @@ def gather_additional_info(info, logdir):
         # elementwise multiplication and sum
         kw_hr_nvidia = np.multiply(time_differences_in_hours, nvidia_power_draw_kw)
 
-    kw_hr_rapl = np.multiply(time_differences_in_hours, power_draw_rapl_kw)
+    kw_hr_rapl = (
```

Comment:
Maybe rather than setting to zero should just not include it, might be misleading.